internal/server/spanner/model.go

```diff
@@ -32,17 +41,77 @@ type Edge struct {
 	Types       []string `spanner:"types"`
 }
 
-// StatVarObservation struct represents a single row in the StatVarObservation table.
-type StatVarObservation struct {
-	VariableMeasured  string `spanner:"variable_measured"`
-	ObservationAbout  string `spanner:"observation_about"`
-	ObservationDate   string `spanner:"observation_date"`
-	Value             string `spanner:"value"`
-	Provenance        string `spanner:"provenance"`
-	ObservationPeriod string `spanner:"observation_period"`
-	MeasurementMethod string `spanner:"measurement_method"`
-	Unit              string `spanner:"unit"`
-	ScalingFactor     string `spanner:"scaling_factor"`
+// Observation struct represents a single row in the Observation table.
+type Observation struct {
+	VariableMeasured  string     `spanner:"variable_measured"`
+	ObservationAbout  string     `spanner:"observation_about"`
+	Observations      TimeSeries `spanner:"observations"`
+	Provenance        string     `spanner:"provenance"`
+	ObservationPeriod string     `spanner:"observation_period"`
+	MeasurementMethod string     `spanner:"measurement_method"`
+	Unit              string     `spanner:"unit"`
+	ScalingFactor     string     `spanner:"scaling_factor"`
+	ImportName        string     `spanner:"import_name"`
+	ProvenanceURL     string     `spanner:"provenance_url"`
+}
+
+type DateValue struct {
+	Date  string
+	Value float64
```

Comment:
We also have non-numeric values for statType=measurementResult stat vars, so maybe we can keep this as a string?

Reply:
I think the api currently only supports floats: https://github.com/datacommonsorg/mixer/blob/3518530571e37f902117e5380f55fdf4930e93c1/proto/stat.proto#L69

So ok to go with this (eventually it gets converted to float anyway) or should we still keep it as string?

Reply:
Oh I see! Yep I think it's fine to keep as a float for now, and then maybe we can revisit later to support non-numeric (I think Ajai mentioned some partner request for this)